### ISSUE TITLE
`APILogicServer create` fails if a foreign key column has the same name as the target table

### ISSUE THREAD

**Issue by charliejharrison** [ApiLogicServer/ApiLogicServer-src#38]
If a foreign key column has the same name as the target table it causes `create` to fail. 

To take an example from the [sample database](https://apilogicserver.github.io/Docs/Sample-Database/): if `Order.CustomerID` was instead named `Order.Customer`, then `create` would throw an SQLAlchemy error: 

```
sqlalchemy.exc.InvalidRequestError: back_populates on relationship 'Customer.OrderList' refers to attribute 'Order.customer' that is not a relationship.  The back_populates parameter should refer to the name of a relationship on the target class.
```

I think this is because when the model generation code tries to declare the relationship in the `Order` model using the target table name, but it finds a name clash with the column so appends a 1 to the name of the relationship. Something like this: 

```
class Order():
    ...
    customer = Column(ForeignKey('Order.customer', ondelete='CASCADE'), nullable=False)
    ...
    customer1 : Mapped["Customer"] = relationship(back_populates=("OrderList"))
```

**Comment by valhuber** [comment]
Thanks for gift-wrapped bug report.  I will save the wrapping.

Will look into it right away.

**Comment by valhuber** [comment]
Fixed in preview:

python -m pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple ApiLogicServer==10.02.00

**Comment by valhuber** [comment]
Fixed in GA: 10.02.00
